<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>GraspCaptureDlgUI</class>
  <widget class="QDialog" name="GraspCaptureDlgUI" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>469</width>
        <height>279</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Grasp Capture</string>
    </property>
    <widget class="Q3GroupBox" name="groupBox1" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>10</y>
          <width>220</width>
          <height>130</height>
        </rect>
      </property>
      <property name="title" >
        <string>Object</string>
      </property>
      <widget class="QLabel" name="textLabel1" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>20</y>
            <width>80</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>Current object:</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="QComboBox" name="objectSelectionBox" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>50</y>
            <width>140</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QPushButton" name="prevObjectButton" >
        <property name="geometry" >
          <rect>
            <x>160</x>
            <y>10</y>
            <width>50</width>
            <height>30</height>
          </rect>
        </property>
        <property name="text" >
          <string>Prev.</string>
        </property>
      </widget>
      <widget class="QPushButton" name="nextObjectButton" >
        <property name="geometry" >
          <rect>
            <x>160</x>
            <y>40</y>
            <width>50</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Next</string>
        </property>
      </widget>
      <widget class="QPushButton" name="clearObjectListButton" >
        <property name="geometry" >
          <rect>
            <x>120</x>
            <y>90</y>
            <width>81</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Clear List</string>
        </property>
      </widget>
      <widget class="QPushButton" name="openObjectFolderButton" >
        <property name="geometry" >
          <rect>
            <x>20</x>
            <y>90</y>
            <width>80</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Open File(s)...</string>
        </property>
      </widget>
    </widget>
    <widget class="QPushButton" name="exitButton" >
      <property name="geometry" >
        <rect>
          <x>399</x>
          <y>239</y>
          <width>60</width>
          <height>31</height>
        </rect>
      </property>
      <property name="text" >
        <string>Exit</string>
      </property>
    </widget>
    <widget class="Q3GroupBox" name="groupBox2" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>150</y>
          <width>220</width>
          <height>120</height>
        </rect>
      </property>
      <property name="title" >
        <string>Capture</string>
      </property>
      <widget class="QLabel" name="textLabel2" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>75</y>
            <width>60</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>Streaming:</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="Line" name="line1" >
        <property name="geometry" >
          <rect>
            <x>106</x>
            <y>12</y>
            <width>20</width>
            <height>101</height>
          </rect>
        </property>
        <property name="frameShape" >
          <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
        </property>
      </widget>
      <widget class="QLabel" name="textLabel3" >
        <property name="geometry" >
          <rect>
            <x>130</x>
            <y>20</y>
            <width>60</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>Total grasps:</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="QLabel" name="numPosesLabel" >
        <property name="geometry" >
          <rect>
            <x>192</x>
            <y>20</y>
            <width>20</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>0</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="QPushButton" name="clearPosesButton" >
        <property name="geometry" >
          <rect>
            <x>130</x>
            <y>70</y>
            <width>81</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Clear List</string>
        </property>
      </widget>
      <widget class="QPushButton" name="savePosesButton" >
        <property name="geometry" >
          <rect>
            <x>130</x>
            <y>40</y>
            <width>81</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Save to File...</string>
        </property>
      </widget>
      <widget class="QPushButton" name="streamingButton" >
        <property name="geometry" >
          <rect>
            <x>63</x>
            <y>70</y>
            <width>40</width>
            <height>30</height>
          </rect>
        </property>
        <property name="text" >
          <string>Start</string>
        </property>
      </widget>
      <widget class="QPushButton" name="instantCaptureButton" >
        <property name="geometry" >
          <rect>
            <x>13</x>
            <y>30</y>
            <width>90</width>
            <height>30</height>
          </rect>
        </property>
        <property name="text" >
          <string>Instant Capture</string>
        </property>
      </widget>
    </widget>
    <widget class="Q3GroupBox" name="groupBox3" >
      <property name="geometry" >
        <rect>
          <x>240</x>
          <y>10</y>
          <width>220</width>
          <height>130</height>
        </rect>
      </property>
      <property name="title" >
        <string>Grasp Information</string>
      </property>
      <widget class="QLabel" name="textLabel5" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>20</y>
            <width>110</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>Number of Contacts:</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="QLabel" name="textLabel6" >
        <property name="geometry" >
          <rect>
            <x>70</x>
            <y>40</y>
            <width>40</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>Quality:</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="QLabel" name="numContactsLabel" >
        <property name="geometry" >
          <rect>
            <x>120</x>
            <y>20</y>
            <width>30</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>0</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="QLabel" name="graspQualityLabel" >
        <property name="geometry" >
          <rect>
            <x>120</x>
            <y>40</y>
            <width>50</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>0.0</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="Q3ProgressBar" name="graspQualityBar" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>80</y>
            <width>200</width>
            <height>31</height>
          </rect>
        </property>
        <property name="percentageVisible" >
          <bool>false</bool>
        </property>
      </widget>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="global" >vector</include>
  </includes>
  <connections>
    <connection>
      <sender>objectSelectionBox</sender>
      <signal>activated(QString)</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>objectSelectionBox_activated(QString)</slot>
    </connection>
    <connection>
      <sender>prevObjectButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>prevObjectButton_clicked()</slot>
    </connection>
    <connection>
      <sender>nextObjectButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>nextObjectButton_clicked()</slot>
    </connection>
    <connection>
      <sender>openObjectFolderButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>openObjectFolderButton_clicked()</slot>
    </connection>
    <connection>
      <sender>clearObjectListButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>clearObjectListButton_clicked()</slot>
    </connection>
    <connection>
      <sender>instantCaptureButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>instantCaptureButton_clicked()</slot>
    </connection>
    <connection>
      <sender>streamingButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>streamingButton_clicked()</slot>
    </connection>
    <connection>
      <sender>savePosesButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>savePosesButton_clicked()</slot>
    </connection>
    <connection>
      <sender>clearPosesButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>clearPosesButton_clicked()</slot>
    </connection>
    <connection>
      <sender>exitButton</sender>
      <signal>clicked()</signal>
      <receiver>GraspCaptureDlgUI</receiver>
      <slot>exitButton_clicked()</slot>
    </connection>
  </connections>
</ui>
